applications/compressible_navier_stokes/couette/tests/couette.output

```diff
@@ -50,6 +50,8 @@ Temporal discretization:
 
 Spatial Discretization:
   Triangulation type:                        Distributed
+  Element type                               Hypercube
```

Comment:
@necioglu Why are the `:` missing here? How did you update the output files for the tests? Did you insert the changes manually?

Reply:
Yes I did add them manually and missed the `:`. Is there an another method? I can change it with a new PR

Reply:
Yes, there is a way to do it quite elegantly. Let discuss this tomorrow in person.

Reply:
Okay. Sorry for this, I thought if the tests passed there shouldn't be any difference in the files, so I didn't double check.